frontend/src/components/AdminTable/index.jsx

```diff
@@ -77,7 +86,7 @@ class AdminTable extends React.Component {
                 </p>
                 <CodebookLabelMenuContainer />
                 <ReactTable
-                    data={admin_data}
```

Comment:
Getting something kind of weird with the admin table pagination? I can't seem to change the number of items to show anymore. Do you have this issue?
<img width="174" alt="Screen Shot 2022-05-24 at 5 06 43 PM" src="https://user-images.githubusercontent.com/10951569/170132586-396ba197-ab77-4ffe-946c-291ee91a875a.png">

Reply:
Fixed